lang/src/main/resources/velocity_template.html

```diff
@@ -1,382 +1,594 @@
5ZMsttxRFo4tx48ZFf8lkEQofk0Ykt6gKFD4m9SS1ZIAtttiice+990Z/yTCdEbxO1MkGGD58ePTXTBah8DFpRHKLqkDhY1IPXk/pbqyQPvaXYToppmsCDBkyJPprJotQ+Jg0IrlFVaDwMamHwscw8VD4ig+Fj0kjkltUBQofk3oofEwdctlllzVWX331n5y/+Lebbrop+iu78JVu8aHwMWlEcouqQOFjUg+Fj6lDunfvLp6/wOccXmONNcSyAKYK4aCNbEPhY9KI5BZVgcLHpB4KH1OHpH0Oo5UP8721DmaC6GFalhdffDH6KyarUPiYNCK5RVWg8DGph8LH1CE8h+sVCh+TRiS3qAoUPib1UPiYOoTncL1C4WPSiOQWVYHCx6QeCh9Th/AcrlcofEwakdyiKlD4mNRThPBh1CT6QrVP9ozvOvHEE5v9p7IOtht9tNpHduJ/49/x39MO+n7hQYbtlEaUYn/gv+Fv6jDhNSb1xnbgmGK72tdFBfh3nAvY5pC+cVUSvrKeB2W4LruCbXcRPtSt6zxr/TtsC7YJ24bzkemsSG5RFSh8TOpJQ/gGDhz4k4cEHmTtN2U8vJZccsnY3yWBm7fLDRo39PaHJ+qE6Tpag4etadmsLrDtaYzGRN1st7sV1B8PPpv9gNVSkoQK3++apP2E78Cx0Z0TXTLd/lkbUF8b2daNonXhnnvuiUqMB3VolweAc6z9nLJNHudBa8pwXfrGVvi6BtdIf9MOzt2k490VbNtuu+0mXkvYd2PHjo3+0j5Jxx3nl095jH0kt6gKFD4m9aQhfEmfb52cFkIi/Y0O3GA/+eSTqAR9kiSutf648eq2VQI3ft+bMqQpDTHBQ9tUB9124fOuLVym4yWVh4cvHpbS37uik0rTOetC0vlt+lHgsj/zPA9aU4br0jc2wgdR1U3Hk0TIuYXWQpdjj+hk2ub+yvhHcouqQOFjUk/WwodfzLYtahK2N0ST8KG1Q/rvNkD6XB9wkEufh1ESqMOIESOi0n8aUwuNq7S6zmsHqUlze0HSahZVEr68z4PWlOG69I1J+HxbkLtIOrfwo8V0brncC3Bd6MoztZYzYZHcoipQ+JjUk7Xw4fPSf3PB5gGnEz7Tw8MGPGBsb8xpCkkrutZGU0uNy4MFLSeuD6m0WvbakV6pV0X4ijgPWpP03Xlel75J45o10f5quyum1liX7Tb90HT9Ica4RXKLqkDhY1JPlsKHB5P0767YvJJMekCnVQdg058PLSdpt3S1gtdp0r4wSZrLa12TPEoPqZDWIh2SaFdB+Io6D1pThuvSN3kIX9KxN3X9sNn3XdG1vGe5/5j/RXKLqkDhY1JPlsIngQc4XnMgeCjavvIy/RJ2EQ5sEwSpK/j/bfpY2ayfatMnCt+P7cb2dwX7BC1n0t+3k9TCYNqPtq0JPg+ppP2PhyMGOeA8aw1enWEfmF5FJ31f1oM2QoWvyPOgK2W4Ln3jInwQWPw9zikE/9f281IrH7bf9FmbH398nVt8JLeoChQ+JvXkJXy4KSc9XHGDNj1ckvrcdMVW+HTlmCTCtC9s+v+YXg3jIWHaF0kSZJIMG2E1PaSS9l/7K11sZ9cDWBc8XE3SZ/NwDT2H2xMifEWfB10pw3XpG1thww+KJOm0aQlOOg5pXEv4oaP7/qxkmVGR3KIqUPiY1JOH8OGhYrq5mW7wpgekjfCZWkRM+8L0gDU9JGxH+Nk8qKT9aXqta/MqytTnKEm+uuqMbbQRtNagNUlX76S+Vq3RfT5v4Sv6POiK6bN5XJe+sRE+1N90rpmOBcqQ6p/GtaT7AWm6lzDpRHKLqkDhY1JPHsJn08nZ1CpiqofpAW37YDK1NunKMLWGuIiQaXuSWlZC66B7SLn0XXKJ6RysmvCV4TxAynBd+sZG+GzOC5vW1iTpNd0LdLJs+t6sWkaZeCS3qAoUPib1ZC18Lg8E3Q02VPhs62BqEUgqx/Qq1FWWTK1eSfsj5FWU6SFl8xrLJ6Zz0DRZLpLWOdgV3/OpLOcBktY+CbkufWMSPhdhMnXVSBJfU2u3rg+e6XWuays44xfJLaoChY9JPWUSPt1D1lROWsJnetD43uBdf9Gb5CvplZBJOHSvE4t6SJnOQZt9p/u8j5T4nk9lOQ+QtPZJyHXpG9N1aNO61xVfeQzZ90W0lDM/jeQWVYHCx6Seqggf0JWTl/AlPWhMrQEuD6iu6OoBkrbJ9CoqaeLYtB9SGJCBvlDYpzgPgFS2iSoJX5nOgzJcl75JU/hMraS67h6m1kGpHqZRvlm1lDM/jeQWVYHCx6QeCl88vg8a0/ebOsdLyUo6pFdYaT6kcE6hJVEqx4cqCV+ZzoMyXJe+SVP4Qu5xPrJo+gxf5+YXyS2qAoWPST0UvniyEj7b72+Nb5k+r3XTeEhBGk119qFOwpfneVCG69I3ZRE+nNO6QTjSKF/dqjN8nZtvJLeoChQ+JvVQ+OKpg/Ahute6WTykbOaN84XC51dmGa5L35RF+BDTQKjWQUV8nVuuSG5RFSh8TOqh8MVTF+EzvdZtH/ka+pAy9XUKgcLnV2YZrkvflEn4TJ9vfa3L17nliuQWVYHCx6QeCl88WQlfnn23ENNr3daHlGmSWdNDyjQyFeA1Mh6GqFd7TOcg+/D51SWtfRJyHvqmTMKH2A6E4uvcckVyi6pA4WNSD4UvnqyEz+UB1RVdPZJWCGiN7Wtd3Ssrm4eU6XtM295JwpfneZDWPknr2nJJmsKXxlQ5pvp0DYTSdWuwmeiaSTeSW1QFCh+Teih88fg+aEyfc+27Y5oDzGa/ml7rdrU26R5Spnqb6mnzMK2T8JXpPEhrn1Rd+EzXgc0xMR0HtGCbWsqTpkNisovkFlWBwsekHgpfPL4PGlPfHd2Ex1JM5dlIkOm1LlYKMB1/0+tc0+dtHsx1Er4ynQdp7ZMyCp+LOJtex9rKo2k/4NhK/w5cjzuTTiS3qAoUPib1UPji8RU+k1wBl1/4PhO+StE
```

Comment:
Please <a href="$vulas-shared-homepage/user/manuals/assess_and_mitigate/">address those dependencies</a> (correct where necessary and exempt where possible) in order to avoid the build exception.